surpriseImpl/svd.py

```diff
@@ -0,0 +1,53 @@
+from surprise import SVD
+from surprise import Dataset
+from surprise.model_selection import cross_validate
+from surprise import accuracy
+from surprise.model_selection import train_test_split
+from dataloaders import movielens_data as loader
+from surprise import Reader
+
+
+def built_in_100k():
+    # Load the movielens-100k data set (download it if needed),
+    data = Dataset.load_builtin('ml-100k')
+
+    return data
+
+
+def custom_pandas_100k():
+    # Load movieLens-100k from pandas dataframe using our data loader
+    movies, ratings = loader.load_data_ml100k()
+
+    # Ratings data is reordered such that it fits Surprise - item, rating, user
+    ratings_frame = ratings[['userId', 'movieId', 'rating']]
+
+    # Requires rating scale. 1-5 in this case.
+    reader = Reader(rating_scale=(1, 5))
```

Comment:
Hvad er en reader 🤔

Reply:
En klasse surprise kræver til at parse filer eller dataframes, den kan feks definere hvilke format hver linje har, men for pandas frames skal man kun bruge en rating scale